provisioning_virtual_machines_and_hosts/_topics/catalogs_and_services.md

```diff
@@ -372,6 +372,125 @@ Service methods have been split based on purpose.
+5. Add a text box named **vm_name**.
+6. Add a dropdown named **size**.
+7. Click the edit pencil icon for the **size** dropdown.
+8. Under **Options** > **Entries** enter the following values:
+   - Enter the following for the first row:
+      - For **Name** field, choose **Large**.
+      - For **Value** field, choose **large**.
+   - Enter the following for the second row:
+      - For **Name** field, choose **Medium**.
+      - For **Value** field, choose **medium**.
+   - Enter the following for the third row:
+      - For **Name** field, choose **Small**.
+      - For **Value** field, choose **small**.
+9. Click **Save**.
+
+#### Create a Service Catalog Item
+
+Use the following steps to create a Service Catalog Item.
+
+1. Click **Service** > **Catalogs**.
+2. Click **Catalogs** > **All Catalogs**.
+3. Click **Configuration** > **Add a New Catalog**.
+4. Provide a **Name** and **Description**:
+   - **Name**: Name displayed in the catalog. For example, `Provision VM`.
+   - **Description**: Desription of the catalog. For example, `Provisioning VM by size`.
+5. Click **Save**.
+6. Click **Catalog Items** > **All Catalog Items** > **Provisioning**.
+7. Click **Configuration** > **Add a New Catalog Item**.
+8. Provide a value for `Catalog Item Type`:
+   - **Catalog Item Type**: The functionality of this catalog item. Choose **VmWare**.
+9. Under the **Basic Information** tab, provide the values for the following fields:
+   - **Name**: Name displayed in the catalog. For example, `Provision VM with Size`.
+   - **Description**: Description of the catalog. For example, `Provision a VM choosing size`.
+   - **Display in Catalog**: Check to display this catalog item.
+   - **Catalog**: Organization concept. For example, `My Company/Provisioning`.
+   - **Dialog**: The dialog created previously. For example, `sm-med-lg dialog`.
+   - **Provisioning Entry Point**:
+     - For **endpoint type**, choose **Embedded Workflow**.
+     - For the **endpoint value** choose the workflow script. For example, `provision-vm-small-med-large.asl`.
+17. Under the **Request Info** > **Catalog** tab enter the following values:
+     - **VM Name**: Name that is overwritten by the **VM name** in the dialog. Enter anything here.
+20. Under the **Environment** tab, choose an appropriate **Host** and **Datastore**:
+    - **Host**: Host that runs the VM.
+    - **Datastore**: Disk that stores the VM.
+22. Under the **Networking** tab, choose an appropriate **Network Adapter**.
+24. Click **Add**.
+
+#### Provisioning a Virtual Machine
+
+1. Click **Services** > **Catalogs** > **Service Catalogs**.
+2. Click **Order** under the newly created service, **Vm with Size**.
+3. Provide the following fields:
+   - **VM name**: Name for the newly created vm. For example `demo-1`.
+   - **size**: The desired VM size. You can choose `small`, `medium` or `large`. For example, choose **small**.
```

Comment:
The presentation strings above in the dialog creation were capitalized, so they should be capitalized here too.

```suggestion
   - **Size**: The desired VM size. You can choose `Small`, `Medium` or `Large`. For example, choose **Small**.
```